drive/client.bal

```diff
@@ -449,9 +449,9 @@ public client class Client {
     # 
     # + fields - The paths of the fields you want included in the response
     # + return - If successful, returns `About`. Else returns `error`
-    @display {label: "Get information about drive"} 
+    @display {label: "Get Information About Drive"} 
```

Comment:
Get Drive Information?

Reply:
This function doesn't have much use case when it comes to low code. So removed the function.